packages/field/src/css/input.ts

```diff
@@ -12,12 +14,17 @@ export default {
     outline: 'none',
     padding: 0,
     width: '100%',
+    height: '38px',
```

Comment:
40px - 2px for vert border

Reply:
Are we not using `box-sizing: border-box`?

Reply:
Hey, yeah, good point.  Why am I doing this? Why is this required? We're using border-box in normalize.

https://github.com/pluralsight/design-system/blob/master/packages/normalize/css/index.css#L20